### ISSUE TITLE
[Feature Request] Obsidian "Re-roll All" Command

### ISSUE THREAD

**Issue by J-MR-T** [valentine195/obsidian-dice-roller#171]
Hey there and thanks so much for your work on this plugin!
I'm currently building a 'DM Screen generator thingy' to generate all kinds of stuff. What I would love (and I would assume wouldn't be too difficult to implement) would be functionality to re-roll all dice on screen. I think this would be best as an obsidian command (although I am fully open to other ways of using this functionality), so something which could be done via Ctrl+P/mapped to a hotkey. This would enable me to just hit a keyboard shortcut to re-roll my entire "Improv" note, containing all the generators for a quick NPC/Encounter/...

Although I am wholly unfamiliar with WebDev/Typescript, if you point me to the right file(s)/mechanisms for re-rolling I might be able to do this too :).

Cheers

**Comment by valentine195** [comment]
This already exists as a command!

**Comment by J-MR-T** [comment]
Oh, sorry, my bad, I couldn't find it earlier. Feel free to close.

**Comment by J-MR-T** [comment]
And now I know why: It does not show up in the command palette:
![image](https://user-images.githubusercontent.com/40471097/226136084-d80d500a-ed7f-4a6f-bb98-c68ee232a2a4.png)
![image](https://user-images.githubusercontent.com/40471097/226136095-2171e650-d90a-47fa-ac7f-b5d7bef385d4.png)

But only in the hotkeys:
![image](https://user-images.githubusercontent.com/40471097/226136115-a27c7018-6985-4f9b-8066-ed0cb70122aa.png)

Idk if that is intentional, but if not you might want to be aware of it :).

**Comment by valentine195** [comment]
Do you have a note open that has dice rollers in it? The command should only appear if that is the case.

**Comment by J-MR-T** [comment]
Yep, I did, but now that you say that I tried it in the same not without live preview, but in the rendered view, there it does show up. Thanks for clearing up the confusion :).

**Comment by valentine195** [comment]
I didn’t consider LP (although this command predates it by quite a bit). I will keep this issue open to add it to that as well.
